packages/ui/src/lib/hooks/useClipboardEvents.ts

```diff
@@ -969,22 +970,27 @@ const handleNativeClipboardPaste = async (
 /** @public */
 export function useMenuClipboardEvents() {
 	const app = useApp()
+	const event = useEvents()
 
 	const copy = useCallback(
 		function onCopy() {
 			if (app.selectedIds.length === 0) return
+
 			handleMenuCopy(app)
+			event('menu', 'copy')
```

Comment:
`event(...)` isn't very clear. The `event` is usually reserved for events. A better name would be `trackEvent(...)` or something like that.

Reply:
`trackEvent` is fine for me, or we could use `onEvent` as that's also the name of the handler?